### DISCUSSION TITLE
paddleOCR V4 版本的识别模型，转不成.nb是什么原因呢？

### DISCUSSION THREAD

**Discussion by jjdadajj** [PaddlePaddle/PaddleOCR#12415] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
  
- 系统环境/System Environment：ubuntu16.04
- 版本号/Version：Paddle：paddle-lite 2.10和2.12都试了  PaddleOCR：V4 官方下载的文件。问题相关组件/Related components：paddle-lite 
- 运行指令/Command Code：
./build.opt/lite/api/opt --model_file=../origin-model/v4/rec/inference.pdmodel --param_file=../origin-model/v4/rec/inference.pdiparams --optimize_out=model/v4/rec --valid_targets=arm --optimize_out_type=naive_buffer

- 完整报错/Complete Error Message：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/95152270/e6040759-cdf7-4f50-8f0f-c464418a8c4c)
jj@jj-VirtualBox:~/Paddle-Lite$ ./build.opt/lite/api/opt --model_file=../origin-model/v4/rec/inference.pdmodel --param_file=../origin-model/v4/rec/inference.pdiparams --optimize_out=model/v4/rec --valid_targets=arm --optimize_out_type=naive_buffer
[W 12/25  0:11:25.375 /home/jj/Paddle-Lite/lite/api/opt.cc RunOptimize:187] Load combined-param model. Option model_dir will be ignored
Loading topology data from ../origin-model/v4/rec/inference.pdmodel
Loading params data from ../origin-model/v4/rec/inference.pdiparams
1. Model is successfully loaded!
[F 12/25  0:11:25.461 ...Lite/lite/model_parser/general/op_desc.h:118 GetAttr] Check failed: it != attrs().end(): No attributes called fix_seed found for dropout
已放弃 (核心已转储)
    

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by chenhao86** [comment]
!paddle_lite_opt --print_model_ops=true --model_dir=output/rec_inference/Student/ --valid_targets=arm  用这个命令判断下 ，模型是否支持  lite  。paddle-lite2.13RC

**Comment by renshujiajia** [comment]
> 1. Model is successfully loaded!
>    [F 12/25  0:11:25.461 ...Lite/lite/model_parser/general/op_desc.h:118 GetAttr] Check failed: it != attrs().end(): No attributes called fix_seed found for dropout

请问这个问题你解决了吗？我遇到类似的问题，我是报错：Check failed: it != attrs().end(): No attributes called beta found for swish
